packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -131,9 +160,12 @@ class MarkdownBuilder implements md.NodeVisitor {
   /// The base directory holding images referenced by Img tags with local or network file paths.
   final String? imageDirectory;
 
-  /// Call when build an image widget.
+  /// Call when building an image widget when no sizing information is available.
```

Comment:
This does not match the code. This callback is called iff `sizedImageBuilder` is not passed; it has nothing to do with whether size information is available at call time.